kartotherian/config.kartotherian.docker.template.yaml

```diff
@@ -16,11 +16,11 @@ logging:
 #    host: logstash1003.eqiad.wmnet
 #    port: 12201
 
-# Statsd metrics reporter
+# Prometheus metrics reporter
 metrics:
-  #type: info
-  #host: localhost
-  #port: 8125
+  type: prometheus
```

Comment:
This wont work unless we are running the latest `service-runner`.

Reply:
We are using the latest service-runner.